scripts/partition_manager/partition_manager.rst

```diff
@@ -38,6 +43,25 @@ Container partitions
 
 The start addresses and sizes of image partitions are used in the preprocessing of the linker script for each image.
 
+RAM partition types
+=====================
+
+Image RAM partition
```

Comment:
```suggestion
Image RAM partitions
```
Or remove the plural for the flash partitions if you think this is misleading for partition types where there can only be one. We should be consistent.